site/gatsby-site/src/tailwind.css

```diff
@@ -544,6 +544,10 @@
         @apply grid grid-cols-1 gap-[6px];
     }
 
+    .tw-header-back-link {
```

Comment:
We should avoid adding these. Best practice is just to add classes to components https://tailwindcss.com/docs/reusing-styles